src/ui/studio/save-creation/index.js

```diff
@@ -352,6 +352,10 @@ const ConnectionUnconfiguredWarning = () => {
 };
 
 const UploadForm = ({ uploadState, handleUpload }) => {
+  const settings = useSettings().upload;
+  const titleField = settings.titleField || FORM_FIELD_REQUIRED;
+  const presenterField = settings.presenterField || FORM_FIELD_REQUIRED;
```

Comment:
Could also use

```suggestion
  const {
    titleField = FORM_FIELD_REQUIRED,
    presenterField = FORM_FIELD_REQUIRED,
  } = useSettings().upload;
```

:relaxed: